### ISSUE TITLE
A11Y - Incorrect voice-over for the the bottom sheet for credit card autofill row item for remember the card flow

### ISSUE THREAD

**Issue by nbhasin2** [mozilla-mobile/firefox-ios#15326]
## Steps to reproduce
* Foxfooding Bug: 6/30/2023 16:15:27
* No particular steps
* Foxfooding Notes
* Try to perform a remember your card flow for credit card
* (a11y low sev, build 9000) The Switch Control is focusing the card info group that is not actionable when tapping (thus, I assume, it's a static element). For elements that do not have any functionality (cannot be tapped or drag-n-dropped etc) it's recommended to ensure they are not focusable, because for Switch Control users who are likely to have limited mobility it would be very cumbersome to first attempt to activate the control (since it's focused with Switch Control in Item mode) and then to return to the previous focusable order (so the focus is again rotating through the drawer and not through the touch options for this element alone). Possibly, ensuring the container view is accessible (read by VoiceOver) while not focusable (no button or other traits on the container and children are hidden from accessibility so Switch Control won't focus it) could be combined into one bug or the Voice Over fix would resolve this issue as well

### Expected behavior
* For elements that do not have any functionality (cannot be tapped or drag-n-dropped etc) it's recommended to ensure they are not focusable, because for Switch Control users who are likely to have limited mobility

### Actual behavior
* The voice over for the Switch Control is focusing the card info group that is not actionable when tapping

### Device & build information
* Device: iPhone
* Build version: 32291
* First seen version: ? 
### Notes
Attachments: 
| Img1      |
| ----------- |
| ![](https://github.com/mozilla-mobile/firefox-ios/assets/8919439/c44202bc-c4cd-48e6-9be3-83d7b2487d22) | 

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6885)

**Comment by data-sync-user** [comment]
➤ Stefan Vladut commented:

fixed as part of FXIOS-6878 ( https://mozilla-hub.atlassian.net/browse/FXIOS-6878 )

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v122 (37590) with iPhone 15 Pro (17.1.2).
